### PR TITLE
qcachegrind - The sourceforge mirror is broken right now

### PR DESCRIPTION
**Pull request by terah**

### CODE REVIEW

File: Library/Formula/qcachegrind.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 class Qcachegrind < Formula
   desc "Visualize data generated by Cachegrind and Calltree"
   homepage 'http://kcachegrind.sourceforge.net/html/Home.html'
-  url 'http://kcachegrind.sourceforge.net/kcachegrind-0.7.4.tar.gz'
+  url 'http://kcachegrind.github.io/files/kcachegrind-0.7.4.tar.gz'
   sha1 'a727e6c39ad2809296dec4ca665f6fa99bda72fb'
```
Comment (bfontaine) [inline Library/Formula/qcachegrind.rb]: Could you please replace this with a `sha256`? Thanks!

File: Library/Formula/qcachegrind.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 class Qcachegrind < Formula
   desc "Visualize data generated by Cachegrind and Calltree"
   homepage 'http://kcachegrind.sourceforge.net/html/Home.html'
-  url 'http://kcachegrind.sourceforge.net/kcachegrind-0.7.4.tar.gz'
+  url 'http://kcachegrind.github.io/files/kcachegrind-0.7.4.tar.gz'
```
Comment (dunn) [inline Library/Formula/qcachegrind.rb]: Can this be `https`?

Should the SF URL be left as a mirror, or vice versa?

Comment (DomT4) [inline Library/Formula/qcachegrind.rb]: This should definitely be `https`.

**Comment by dunn** [comment]
The SF mirror is back up. Could you update this to have SF and GitHub, with one as primary `url` and the other as `mirror`, and bump the `sha1` to `sha256`? Thanks so much!

**Comment by DomT4** [comment]
Ping.

**Comment by DomT4** [comment]
Looks like you've moved this to #42439, Closing.

**Comment by terah** [comment]
Yep.. done

**Pull request closed without merging**